dart/test/code_area_test.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:open_location_code/open_location_code.dart' as olc;
```

Comment:
I don't see the purpose of this test, as CodeArea is just a model. We don't have tests for CodeArea in any of the other implementations. Could you revert?

Reply:
Did it.
Thanks!